tests/accelerators/test_accelerator_connector.py

```diff
@@ -445,18 +445,20 @@ def test_accelerator_choice_multi_node_gpu(
     assert isinstance(trainer.strategy, plugin)
 
 
-@pytest.mark.skipif(torch.cuda.is_available(), reason="test doesn't require GPU")
-def test_accelerator_cpu():
+@mock.patch("torch.cuda.is_available", return_value=False)
+def test_accelerator_cpu(mack_gpu_avalible):
```

Comment:
Since it's unused (also a typo)

```suggestion
def test_accelerator_cpu(_):
```